driver/kubernetes/factory.go

```diff
@@ -46,18 +69,50 @@ func (*factory) Priority(ctx context.Context, endpoint string, api dockerclient.
 }
 
 func (f *factory) New(ctx context.Context, cfg driver.InitConfig) (driver.Driver, error) {
-	if cfg.KubeClientConfig == nil {
-		return nil, errors.Errorf("%s driver requires kubernetes API access", DriverName)
+	var err error
+	var cc ClientConfig
+	if f.cc != nil {
+		cc = f.cc
+	} else {
+		cc, err = ctxkube.ConfigFromEndpoint(cfg.EndpointAddr, cfg.ContextStore)
+		if err != nil {
+			// err is returned if n.Endpoint is non-context name like "unix:///var/run/docker.sock".
+			// try again with name="default".
+			// FIXME(@AkihiroSuda): n should retain real context name.
```

Comment:
What was "n"? 

Maybe the variable was renamed since then? (hard to check from phone now)

Reply:
Seems it was the node https://github.com/docker/buildx/blob/63073b65c01c415db8b34053ac3f0b7f2c14d27a/commands/util.go#L90

As we now have `ContextStore` in driver `InitConfig` maybe it would be fine to have `ContextName` as well and use it here?